src/core/templates/ResourceMembers.ts

```diff
@@ -26,7 +26,7 @@ export class ResourceMembers extends BaseService {
 
     if (includeInherited) url.push('all');
 
-    return RequestHelper.get(this, url.join('/'), { options });
+    return RequestHelper.get(this, url.join('/'), { ...options });
```

Comment:
instead of unpacking, I should we just toss the `options` object at it.
```suggestion
    return RequestHelper.get(this, url.join('/'), options);
```